bot/cogs/core/voting.py

```diff
@@ -4,13 +4,71 @@
 import arrow
 import discord
 from discord.ext import commands
+from discord.utils import format_dt
+
 
 from bot.cogs.core.database import Database
 from bot.cogs.core.quests import Quests
+from bot.utils.misc import get_user_and_lang_from_loc
 from bot.villager_bot import VillagerBotCluster
 from common.models.topgg_vote import TopggVote
 
 
+class VoteReminderView(discord.ui.View):
+    def __init__(
+        self,
+        *,
+        bot: VillagerBotCluster,
+        user: discord.User,
+        timeout: float = 300.0,
+    ):
+        super().__init__(timeout=timeout)
+
+        self._bot = bot
+        self._user = user
+
+        self.message: discord.Message | None = None
+
+    @property
+    def _db(self) -> Database:
+        return typing.cast(Database, self._bot.get_cog("Database"))
+
+    async def interaction_check(self, interaction: discord.Interaction) -> bool:
+        return interaction.user.id == self._user.id
+
+    @discord.ui.button(label="Add Vote Reminder", style=discord.ButtonStyle.gray)
+    async def btn_add_vote_reminder(
+        self, interaction: discord.Interaction, button: discord.ui.Button
+    ):
+        await interaction.response.defer(thinking=False)
+        _, lang = get_user_and_lang_from_loc(self._bot.l, self._user)
```

Comment:
```suggestion
        lang = self._bot.l['en']
```

Unfortunately there's no per-user language setting, I think this will work